test/e2e/explore.test.js

```diff
@@ -119,19 +138,20 @@ test.describe('Explore screen', () => {
     })
 
     test('should open dag-pb', async ({ page }) => {
+      test.setTimeout(60000)
```

Comment:
hm.. probably shot in the dark, but iirc in IPLD explorer  if local gateway is not available, we fallback to public one, so perhaps the slowness is because local gateway cant be used, and tests pass only on slow fallback?